docs/01_getting-started/03_go-to-the-next-level/06_forms.md

```diff
@@ -56,19 +56,73 @@ Define the `insertTrade` function in the file **home.ts**:
   }
 ```
 
+The form is configured as required, but if you look at the page currently you will not see it. Next we need to add some sections and styling to the page. Ensure the template in **home.template.ts** looks like the following.
+
+```html {1,15-19} title='home.template.ts'
+<div class="column-split-layout">
+	<zero-grid-pro persist-column-state-key="position-grid-settings">
+		<grid-pro-genesis-datasource
+			resource-name="ALL_POSITIONS"
+			order-by="INSTRUMENT_ID"
+		></grid-pro-genesis-datasource>
+		${repeat(
+			() => positionColumnDefs,
+			html`
+				<grid-pro-column :definition="${(x) => x}"></grid-pro-column>
+			`
+		)}
+		<grid-pro-column :definition="${(x) => x.singlePositionActionColDef}"></grid-pro-column>
+	</zero-grid-pro>
+	<foundation-form
+		resourceName="EVENT_TRADE_INSERT"
+		@submit=${(x, c) => x.insertTrade(c.event as CustomEvent)}
+	></foundation-form>
+</div>
+```
+
+And then set the styling required on the `host` element, and the `column-split-layout` div in **home.styles.ts**.
+```typescript title='home.styles.ts'
+import { css } from '@microsoft/fast-element';
+import { mixinScreen } from '../../styles';
+
+export const HomeStyles = css`
+  :host {
+    ${mixinScreen('flex')}
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  .column-split-layout {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    width: 100%;
+  }
+
+  .row-split-layout {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    width: 100%;
+    height: 50%;
+  }
+`;
+```
+
 After refreshing your application, a form should be displayed. The form might sit on top of the grid or by itself, depending on whether you appended to or replaced the already existing xml in **home.template.ts**.
```

Comment:
any thoughts on the word `xml` here? how about `markup`?

Reply:
Done